lambda_uploader/package.py

```diff
@@ -103,6 +103,16 @@ def package(self):
 
         shutil.copytree(os.path.join(self._pkg_venv, site_packages),
                         package)
+        
+        lib64_site_packages = 'lib64/python2.7/site-packages'
+        if sys.platform == 'win32' or sys.platform == 'cygwin':
```

Comment:
I think this is an unnecessary if statement.  You should be able to set lib64_site_packages right after the site_packages declaration.